pkgs/applications/graphics/kcc/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, mkDerivationWith, wrapQtAppsHook, python3Packages, p7zip, p7zipSupport ? true
```

Comment:
Same here

```suggestion
{ lib
, mkDerivationWith
, wrapQtAppsHook
, python3Packages
, p7zip
, p7zipSupport ? true
```

Reply:
Also maybe use something like `archiveSupport` or `cbxSupport` instead of `p7zipSupport`. The software used for creating those files should be just an implementation detail.